hazelcast/src/test/java/com/hazelcast/client/protocol/compatibility/ReferenceObjects.java

```diff
@@ -927,4 +928,8 @@ private static boolean isEqualStackTrace(StackTraceElement stackTraceElement1, S
     public static SqlError anSqlError = new SqlError(anInt, aString, aUUID, aBoolean, aString);
     public static SqlPage aSqlPage = SqlPage.fromColumns(Collections.singletonList(SqlColumnType.INTEGER), Collections.singletonList(Arrays.asList(1, 2, 3, 4)), true);
     public static HazelcastJsonValue aHazelcastJsonValue = new HazelcastJsonValue("{'value': ''}");
+    public static List<PartitioningAttributeConfig> aListOfPartitioningAttributeConfigs = Collections.singletonList(
+            new PartitioningAttributeConfig(aString)
+    );
+    public static PartitioningAttributeConfig aPartitioningAttributeConfig = new PartitioningAttributeConfig("attr1");
```

Comment:
This is not needed for now, and it would probably not work in the future because it is not using `aString` as its argument and there is no override in the protocol code generator. I would suggest removing it altogether.